queue_services/entity-emailer/src/entity_emailer/worker.py

```diff
@@ -142,6 +143,20 @@ def process_email(email_msg: dict, flask_app: Flask):  # pylint: disable=too-man
         elif etype and etype == 'bc.registry.bnmove':
             email = bn_notification.process_bn_move(email_msg, token)
             send_email(email, token)
+        elif etype and etype == 'bc.registry.dissolution':
+            email = involuntary_dissolution_stage_1_notification.process(email_msg, token)
+            # Confirm the data.furnishingName
+            furnishing_name = email_msg['data']['furnishing']['furnishingName']
+            if furnishing_name not in involuntary_dissolution_stage_1_notification.PROCESSABLE_FURNISHING_NAMES:
+                raise QueueException('Furnishing name is not valid.')
+            try:
+                send_email(email, token)
+                # Update corresponding furnishings entry as PROCESSED
+                involuntary_dissolution_stage_1_notification.post_process(email_msg, 'PROCESSED')
+            except Exception as _:  # noqa B902; pylint: disable=W0703
+                # Update corresponding furnishings entry as FAILED
+                involuntary_dissolution_stage_1_notification.post_process(email_msg, 'FAILED')
+                raise
```

Comment:
https://github.com/bcgov/lear/pull/2823#discussion_r1678369855

Update corresponding furnishings entry as “PROCESSED” or “ERROR” depending on HTTP status that comes back from notify api